app/src/main/kotlin/cz/covid19cz/erouska/exposurenotifications/ExposureNotificationsRepository.kt

```diff
@@ -311,14 +311,23 @@ class ExposureNotificationsRepository @Inject constructor(
                 accepted = false
             )
         })
-        val latestExposure = db.dao().getLatest().firstOrNull()?.daysSinceEpoch
-        val lastNotifiedExposure = db.dao().getLastNotified().firstOrNull()?.daysSinceEpoch
-        if (latestExposure != null && latestExposure != lastNotifiedExposure) {
+
+        val latestExposure = db.dao().getLatest().firstOrNull()
+        val latestExposureTime = latestExposure?.daysSinceEpoch
+        val lastNotifiedExposureTime = db.dao().getLastNotified().firstOrNull()?.daysSinceEpoch
+        val latestExposureTimestamp = latestExposure?.importTimestamp ?: 0L
+
+        val appOpenedTimestamp = prefs.getLastTimeAppOpened()
+        val appOpenedAfterLastNotification = latestExposureTimestamp < appOpenedTimestamp
```

Comment:
Jenom si chci být jistý, že to správně chápu - to `latestExposureTimestamp` je čas kdy přišla poslední notifikace? Ne kdy jsem se potkal s nakaženým?

Reply:
ak som to pochopila spravne, je to cas, kedy sa exposure ulozila do db (par riadkov nad tym, rovnaka metoda). o exposure sa notifikuje iba ak sa stiahla a teda aj ulozila, takze mi ten timestamp na toto pouzitie dava zmysel